frontend/src/scenes/experiments/experimentLogic.test.ts

```diff
@@ -88,12 +88,12 @@ describe('experimentLogic', () => {
     describe('selector values', () => {
         it('given an mde, calculates correct sample size', async () => {
             await expectLogic(logic).toMatchValues({
-                minimumDetectableChange: 5,
+                minimumDetectableChange: 1,
             })
 
-            expect(logic.values.minimumSampleSizePerVariant(20)).toEqual(1024)
+            expect(logic.values.minimumSampleSizePerVariant(20)).toEqual(25600)
 
-            expect(logic.values.minimumSampleSizePerVariant(40)).toEqual(1536)
+            expect(logic.values.minimumSampleSizePerVariant(40)).toEqual(38400)
```

Comment:
Just highlighting this - MDE changed from 5 to 1, so this increase is expected

Reply:
is mde 1 because no conversion rate / count data present?

Reply:
correct